.github/workflows/test.yml

```
@@ -110,13 +110,94 @@ jobs:
       - name: Unit Tests
         run: sbt ++${{ matrix.scala }} standardLibrary/test
 
+  website-api-docs:
+    name: API Docs
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        project: [chisel3]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.8
+      - name: Cache Scala
+        uses: coursier/cache-action@v5
+      - name: Calculate API Docs Cache Key
+        id: calc-key
+        run: |
+          KEY=$(make --always-make --dry-run apis-${{ matrix.project }} | sha256sum | awk '{print $1}')
```

Is this supposed to be:

```suggestion
          KEY=$(make -C website --always-make --dry-run apis-${{ matrix.project }} | sha256sum | awk '{print $1}')
```

There's a failure in that CI action step which was surprising.

---

ah, yeah this is exactly what i was worried about -- we keep hittin gin the cache so haven't tested this. I need to manually flush this cache i think to make sure it works even on a cache miss